tests/cephfs/cephfs_utilsV1.py

```diff
@@ -2579,6 +2580,120 @@ def dd():
         f()
         return f
 
+    def run_ios_new(self, client, mounting_dir, io_tools=["dd", "smallfile"], **kwargs):
+        def smallfile():
+            log.info("IO tool scheduled : SMALLFILE")
+            io_params = {
```

Comment:
Keep the default config along with this can we add io_params to kwargs so that user has felxibility in adjusting the io_params.
We can rename for each tool io_params --> smallfile_io_params, dd_io_params ..